### DISCUSSION TITLE
Plotting of Schematics

### DISCUSSION THREAD

**Discussion by zmladen** [Eomys/pyleecan#713] [General]
Hello,

this is connected to the creation of the new slot SlotW31B. When plotting the schematics the plot shows the devided active surfaces and the whole active surfaces (orange color). The opening surfaces are plotted as wedges.

![image](https://github.com/user-attachments/assets/dc0b5a02-c781-4177-9f50-015037bf6035)

 How to hide the "whole active surface" to get something like this for the schematics?

![image](https://github.com/user-attachments/assets/c3ddd209-582b-45f4-b763-cd70aeba56c9)

I got this plot by commenting the line:

`
    # Add the magnet to the fig
    for patch in patches:
        ax.add_patch(patch)
`
in plot_active.py method of the slot class.

Thanks,
Mladen

**Comment by BonneelP** [comment]
Hello,

The "whole active surface" is the full surface without the air around (both layer + "i_tan" rectangle) ? I guess you can take inspiration from SlotW60 and SlotW61 that have a similar configuration.

Best regards,
Pierre
